### ISSUE TITLE
Support option to ignore weekends (or do this by default)

### ISSUE THREAD

**Issue by acquawork** [google/ezgantt#1]
Currently, the time estimates include the weekend which is probably not something that should be done by default. It would be nice if the default behavior was to ignore Sat / Sun and optionally allow weekends to be included.

**Comment by salamanders** [comment]
What sort of durations are you using?  Nd?  Nw?  Nm?

**Comment by acquawork** [comment]
I'm using days currently.

**Comment by salamanders** [comment]
Can you try weeks?  I think it would do the math of 7 days to a week, then it would take into account weekends.  If you are committed to days as the time estimates, then yes, agree that FR would be useful to not slot days into weekends.

**Comment by acquawork** [comment]
Well, we're trying to do planning for the next two weeks so it would be
pretty cumbersome to use weeks for each task.

**Comment by salamanders** [comment]
Gotcha.  Cool, will see if I can do that!

**Comment by zupolgec** [comment]
See https://github.com/google/google-visualization-issues/issues/2274

**Comment by salamanders** [comment]
I agree with @zupolgec that this should be fixed at the Gantt-chart level, please upvote google/google-visualization-issues#2274

**Comment by vchudnov-g** [comment]
I would suggest reading any number of cells in the first column with a format like this, for example:

```
#exclude: Sat,Sun
#exclude: 2017-12-25
#exclude: 2017-11-23,2017-11-24
#exclude: 2018-01-02/2018-01-05
```
The above shows recurring days of the week, a specific date, multiple dates, and a date range, respectively.

This allows the team to specify their regular days off (not everyone uses Sat and Sun) as well as special times (holidays, vacations).

In a future iteration, it might be helpful to _also_ allow per-person exclusions, if only one person has a day off.  Something like `#exclude:alice:2017-12-26`

**Comment by salamanders** [comment]
Open to pull requests for this!

**Comment by salamanders** [comment]
I think this would have to be fixed in google/google-visualization-issues#2274
